gems/pending/util/xml/miq_rexml.rb

```diff
@@ -144,7 +144,16 @@ def initialize(first, second = nil, parent = nil)
         self.name = first
         # This is the old method that does not handle HTML encoded strings properly
         # @value = second.to_s
-        @value = Text.unnormalize(second.to_s, nil)
+        begin
+          @value = Text.unnormalize(second.to_s, nil)
+        rescue => err
+          if err.class == ::Encoding::CompatibilityError
+            second_utf8 = second.to_s.force_encoding('UTF-8')
+            @value = Text.unnormalize(second_utf8, nil)
+          else
+            $log.error "Encoding error: #{second_utf8}" if $log
+          end
+        end
```

Comment:
@hsong-rh Can you create a test case for to test this against the rexml from ruby's standard library?

Note: This monkey patched version came from ruby 1.8.6 or so.  See the comment on line 135 about ruby 1.8.6 patch 111.  Are we planning on moving to nokogiri here?  It worries me that we still use rexml.  rexml is nice for test code but can't handle much more than tiny xmls.  In addition, we have a forked version checked in.  😢